'

```diff
@@ -0,0 +1,89 @@
+# CloudTrail
+
+resource "aws_cloudtrail" "data_refinery_s3_cloudtrail" {
+  name                          = "data-refinery-s3-cloudtrail-${var.user}-${var.stage}"
+  depends_on                    = ["aws_s3_bucket.data_refinery_cloudtrail_logs_bucket"]
```

Comment:
Isn't this implied when you reference the id of this resource in the line below?

You don't need to retest it over just this, so you can leave it, but it's just a thought.